apps/sms/views/shared/js/bootstrap.js

```diff
@@ -1,5 +1,14 @@
-/* global Startup */
+/* global App,
+          Startup */
 
+(function() {
 'use strict';
 
+var shimFrame = document.createElement('iframe');
```

Comment:
Yeah I think it definitely worth a try with static iframe and see if we could get app_instance_id directly instead of via url.